### ISSUE TITLE
The context menu key does not work

### ISSUE THREAD

**Issue by bmivzkrp** [getsolus/brisk-menu#44]
After focusing on any program, press the context menu key. You will notice that it does not work. You need to use the right mouse button, but this is not very convenient for users of the Orca screen reader.
Please make the context menu key work in the same way as the right mouse button.
I will be very grateful for your attention to this problem.
